controlpanel/api/models/app.py

```diff
@@ -51,16 +51,23 @@ def customers(self):
     def add_customers(self, emails):
         emails = list(filter(None, emails))
         if emails:
-            auth0.AuthorizationAPI().add_group_members(
-                group_name=self.slug,
-                emails=emails,
-                user_options={"connection": "email"},
-            )
+            try:
+                auth0.AuthorizationAPI().add_group_members(
+                    group_name=self.slug,
+                    emails=emails,
+                    user_options={"connection": "email"},
+                )
+            except auth0.Auth0Error as e:
+                raise AddCustomerError(e)
 
     def delete_customers(self, user_ids):
```

Comment:
Again, it feels this `delete_customers()` method belongs in `cluster.App` rather than in the model.

Reply:
When I say "feels" I mean it seems logical to as it interacts with the cluster rather than with the DB.

Reply:
I can't refactor everything at once! (or so I'm told)

Reply:
yeah sorry, it was just an observation more than an incitation to move even more things around, my bad 😁